src/stylesheets/elements/_buttons.scss

```diff
@@ -349,10 +392,10 @@
   height: 45px;
   min-width: 225px;
```

Comment:
Should we continue the uswds min width util classes and add:

//USWDS
.minw-15 120px

//-- SDS Below
.minw-16 240px
.minw-17 480px
etc

Reply:
I did not touch that code

Reply:
fixed